05_casts/05_00_Interesting_cases/05_00_00_up_and_down_casts.cpp

```diff
@@ -0,0 +1,44 @@
+// Show how static_cast works, and also that there is no check in it,
```

Comment:
Where is the example with `dynamic_cast`? The description for all files is not clear